spec/requests/cinema_halls_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+
+# This spec was generated by rspec-rails when you ran the scaffold generator.
+# It demonstrates how one might use RSpec to test the controller code that
+# was generated by Rails when you ran the scaffold generator.
+#
+# It assumes that the implementation code is generated by the rails scaffold
+# generator. If you are using any extension libraries to generate different
+# controller code, this generated spec may or may not pass.
+#
+# It only uses APIs available in rails and/or rspec-rails. There are a number
+# of tools you can use to make these specs even more expressive, but we're
+# sticking to rails and rspec-rails APIs to keep things simple and stable.
+
+RSpec.describe "/cinema_halls", type: :request do
+  
+  # CinemaHall. As you add validations to CinemaHall, be sure to
+  # adjust the attributes here as well.
+  let(:valid_attributes) {
+    skip("Add a hash of attributes valid for your model")
```

Comment:
this should be adjusted. You can use `FactoryBot#attributes_for(:cinema_hall)` here. This methods is overridable so one can do `attributes_for(:cinema_hall, name: "")` if you'd like invalid attributes!